lib/rentlinx/validators/attribute_processor_service.rb

```diff
@@ -0,0 +1,31 @@
+module Rentlinx
+  class AttributeProcessor
+    attr_reader :errors
+    attr_reader :processed_attrs
+
+    def initialize(attrs)
+      @attrs = attrs
+      @processed_attrs = @attrs
+      @errors = {}
+    end
+
+    def process
+      validators = { phoneNumber: PhoneValidator,
```

Comment:
Same question about requiring file here. By looking just at this file, I have no idea where these validators come from and it is somewhat confusing.